### DISCUSSION TITLE
24-hour time

### DISCUSSION THREAD

**Discussion by caspervandenberg** [manuelernestog/weektodo#288] [Ideas]
Could you please add the option to use a 24-hour time format? It would be really helpful. Thank you!
![image](https://github.com/user-attachments/assets/e8fcc7f5-be8e-45d2-872f-b1a1faf7fdf7)

**Comment by Nels-J** [comment]
Today on earth the market share between usage of am/pm Vs 24hours is near to 50/50 % it would be appreciate lot of users indeed. Thanks anyway on this tool really user friendly !
